### ISSUE TITLE
Check CodeClimate [1h]

### ISSUE THREAD

**Issue by AronNovak** [Gizra/drupal-elm-starter#219]
https://github.com/Gizra/og/pull/505#issuecomment-492108145

**Comment by AronNovak** [comment]
@amitaibu 

TL;DR: I'd not invest in integrating CodeClimate.

I played with CodeClimate previously and I am not really convinced.
https://github.com/Gizra/og/pull/505#issuecomment-494151901

> CodeClimate will allow us to "confirm" issues and fix them in new issues.

This is not really needed, only for legacy projects, but at those, usually we do not aim to clean up all codebase.

So for now, Drupal + DrupalPractice enforcing via Travis, how Elm Starter and all other project does, seem to work well. The duplicated code detector is nice, but due to the nature of the Drupal codes, it would be quite hard to eliminate the false positives, so with or without CodeClimate, human review is essential.

**Comment by amitaibu** [comment]
:+1:
